contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -777,16 +731,17 @@ handle_where_clause_restargets_left(ParseState *pstate, Node *w_clause, List *ex
 					if (attrno == InvalidAttrNumber)
 					{
 						ereport(ERROR,
-						(errcode(ERRCODE_UNDEFINED_COLUMN),
-						 errmsg("column \"%s\" of relation \"%s\" does not exist",
-								name,
-								RelationGetRelationName(pstate->p_target_relation))));
+								(errcode(ERRCODE_UNDEFINED_COLUMN),
```

Comment:
nit: unnecessary diff. 

Also, is this case different than the others? ie throwing an error rather than returning null result set?

Reply:
Updated the unnecessary change in latest rev
Kept it in two separate if intentionally as this is a helper function hence it can be called by some other function. Then when the outer if satisfies, we do not know what will be the behaviour at that time. So, better to put extra check for sp_describe_undeclared_parameters.